build/Dockerfile

```
@@ -31,11 +31,8 @@ ENV PIP_PACKAGES="\
     ansible[azure] \
     virtualenv \
     molecule \
```

Maybe even https://github.com/ansible/molecule/blob/master/setup.cfg#L44
```suggestion
     molecule[azure] \
```
but it'll work only after we release a new dist

---

So merge it for now, and foresee a next change? 
Maybe define something like an `all_extras` extra installing azure,docker,ec2 etc...so that this complete list could be replaced by a single molecule[all_extras]?

---

I don't like `all_extras` thing because it needs programming and I want to keep dist meta declarative + it's a user-facing feature and they would rarely need all envs support, this'd implicitly require them to have build pre-requisites for a bunch of unrelated staff + I'd be non-deterministic in terms of features it enables across versions.